benchto-driver/src/main/java/io/trino/benchto/driver/loader/BenchmarkLoader.java

```diff
@@ -74,7 +74,8 @@
 
     private static final String BENCHMARK_FILE_SUFFIX = "yaml";
 
-    private static final int DEFAULT_RUNS = 3;
+    private static final int DEFAULT_RUNS = 1;
+    private static final int DEFAULT_QUERY_RUNS = 6;
```

Comment:
Can we keep the default for runs? This will be a breaking change otherwise.
```suggestion
    private static final int DEFAULT_RUNS = 3;
    private static final int DEFAULT_QUERY_RUNS = 1;
```

Reply:
It is breaking change regards to the current state. It shoud be breaking change - we acutally want to restore old approach when query is repeated to be measured, we do not want to repeat whole benchmark by default.